src/typeDefs/types.ts

```diff
@@ -229,6 +236,8 @@ export const types = gql`
     When paginating backwards, are there more items?
     """
     hasPreviousPage: Boolean!
```

Comment:
Add `hasNextPage` here, and please explain what `totalPages` `nextPageNo` and `prevPageNo` mean here? Connections don't work like page based pagination or am I missing something here?

Reply:
@xoldyckk 
Actually I had created a combined `PageInfo` type, which would contain attributes for both cursor based navigation (as we are implementing) and Offset based pagination which already exists in the codebase.

Though in hindsight, this is a practice, and I have defined a new type `CursorPageInfo` which only contains the attributes specific to the same.